app/javascript/src/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,43 @@
+$font-primary: #464646;
```

Comment:
all colours are defined in https://github.com/3scale/porta/blob/dashboard-links/app/assets/stylesheets/provider/_colors.scss#L1. Try to pick one that makes sense instead of adding a new color.